converters/fromZigbee.js

```diff
@@ -1643,20 +1643,62 @@ const converters = {
                     default:
                         // if (meta.logger) meta.logger.debug(`plug.mmeu01: unknown vtype=${data[i+1]}, pos=${i+1}`);
                     }
-                    payload[index] = value;
-                    // if (meta.logger) meta.logger.debug(`plug.mmeu01: recorded index ${index} with value ${value}`);
+                    switch (index) {
+                    case 3:
+                        // temperature
+                        payload.temperature = calibrateAndPrecisionRoundOptions(value, options, 'temperature');
+                        break;
+                    case 100:
+                        // state
+                        payload.state = value === 1 ? 'ON' : 'OFF';
+                        break;
+                    case 149:
+                        // consumption
+                        payload.consumption = precisionRound(value, 2);
+                        break;
+                    case 150:
+                        // voltage
+                        payload.voltage = precisionRound(value * 0.1, 1);
+                        break;
+                    case 151:
+                        // current
+                        payload.current = precisionRound(value * 0.001, 4);
+                        break;
+                    case 152:
+                        // power
+                        payload.power = precisionRound(value, 2);
+                        break;
+                    default:
+                        // if (meta.logger) meta.logger.debug(`plug.mmeu01: unknown index $(index) with value ${value}`);
+                    }
+                    if (meta.logger) meta.logger.debug(`plug.mmeu01: recorded index ${index} with value ${value}`);
```

Comment:
I don't think we should always log here. maybe only in default case?

Reply:
Same goes for all logging below e.g. ` if (meta.logger) meta.logger.debug(`plug.mmeu01: attr. 513 (pwr_out_mem) has value ${msg.data['513']}`);`

Reply:
Ok